scripts/run_all.sh

```diff
@@ -26,9 +26,9 @@
 set -xu
 
 GEM5=./../build/ALL/gem5.opt
-GEM5_CONFIG=./gem5-configs/fs-fdp.py
+GEM5_CONFIG=./gem5-configs/fs-multiarch.py
 
-ARCH="amd64"
+ARCH="arm64"
```

Comment:
```suggestion
ARCH="amd64"
```
I suggest using as default x86 as most people might not have an Arm machine